core/src/main/scala/org/http4s/StaticFile.scala

```diff
@@ -1,18 +1,21 @@
 package org.http4s
 
+import java.io._
+import java.net.URL
+
+import cats.Semigroup
 import cats.data._
 import cats.effect._
-import cats.implicits.{catsSyntaxEither => _, _}
+import cats.implicits._
```

Comment:
That catsSyntaxEither trick actually is necessary with the strict compiler flags and 2.11 support.

Reply:
Got it, thanks!